README.md

```diff
@@ -65,7 +65,7 @@ the `data` directory; you can access it at `/data/<filename>.json`.
 
 ### Available dependencies
 
-The repo includes jQuery, Backbone, Underscore, and RequireJS. If there's other
+The repo includes jQuery, Backbone, and Underscore. If there's other
 stuff you'd find useful, you can put it in the `lib` directory.
```

Comment:
might as well get rid of the idea of a `lib` directory since one does not exist.